packages/catalog-model/src/kinds/GroupEntityV1alpha1.test.ts

```diff
@@ -172,4 +172,26 @@ describe('GroupV1alpha1Validator', () => {
     (entity as any).spec.children = [];
     await expect(validator.check(entity)).resolves.toBe(true);
   });
+
+  // members
+
+  it('accepts missing members', async () => {
```

Comment:
these tests don't test the happy path! :)

so i suggest adding a `members: ['test']` or whatever to the entity at the top, that then gets mutated / replaced in these individual tests - that's how the other tests above work